apps/closures-and-restrictions-map/src/components/Detail/DynamicDetail.tsx

```diff
@@ -48,6 +50,25 @@ export const DynamicDetail: React.FC<IDetail> = ({ featureProps, streetViewUrl }
 
     const { data: attachments } = useArcgisAttachments(DIGUPS_URL, objectId || 0);
 
+    const { data: disordersAttachments } = useArcgisAttachments(
+        DISORDERS_URL,
+        objectId || originalProperties?.objectid || 0,
+    );
+
+    const getDisorderImgUrlList = (): Array<string> => {
```

Comment:
it's a bit strange to create a fn which is then used in exactly one place but called on each render

would prefer writing it in a way of:

```
let imageUrlList: string[] = []
if (layer === 'disorders' && disordersAttachments?.length > 0) {
  imageUrlList = disordersAttachments.filter(a => a?.keywords === 'foto_havarie').map(a => `${DISORDERS_URL}/${originalProperties?.objectid}/attachment/${a.id}`)
}
```